src/components/Node/Node.module.scss

```diff
@@ -28,6 +28,10 @@
     background-color: $red;
   }
 
+  &.declined {
+    background-color: #404040;
```

Comment:
I'm not sure if this color is used elsewhere, but it would be better to keep this color with the rest in `_src/styles/_variables.scss` sooner than later.

Maybe could call it `$black-medium`?